satpy/readers/seviri_l1b_hrit.py

```diff
@@ -15,7 +15,8 @@
 #
 # You should have received a copy of the GNU General Public License along with
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
-r"""SEVIRI HRIT format reader.
+"""SEVIRI HRIT format reader
+============================
```

Comment:
Why is this being reverted? Same question with the `Arguments` thing below. These were specifically changed by me to avoid warnings and errors when generating the sphinx documentation.

FYI @mraspaud sphinx doesn't like the section header at the top of the docstring.

Reply:
I changed these as Flake8 was complaining about them. As it causes a problem I will revert.

Reply:
These issues don't seem to have been resolved. There no `r` at the beginning of the string, the `===` still exists in the second line, and the `Reader ` has still been removed from the `Arguments` section below.